src/styles/blocks/cards.scss

```
@@ -0,0 +1,133 @@
+main {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+  grid-template-rows: repeat(2, auto);
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ....;
```